pkg/engine/destroy.go

```diff
@@ -80,8 +80,7 @@ func newDestroySource(
 
 	// Like Update, if we're missing plugins, attempt to download the missing plugins.
 
-	if err := ensurePluginsAreInstalled(ctx, plugctx.Diag, plugins.Deduplicate(),
-		plugctx.Host.GetProjectPlugins()); err != nil {
```

Comment:
Why aren't we passing `plugctx.Host.GetProjectPlugins()` anymore?

Reply:
ah, all fixed up now